type_check_failed,error_msgs
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 19, in parse_config_file: No attribute 'read' on List[nothing] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in set_global_options: Name 'setter' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 6, in reset_global_options: Name 'setter' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in error400: Name 'e' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in __init__: Can't assign attribute 'datacenter_name' on Dict[str, Any] [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in get_all_episodes: Name 'r' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 10, in get_episode: No attribute 'create_episode_from_row' on str [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in get_movie: No attribute 'create_movie_from_row' on str [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in clear_movie: Name 'c' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in write_metadata: Name 'meta_file_path' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in test_subscription_create: Name 'response' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in test_subscription_cancel_page_without_subscription: Name 'response' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in __repr__: Name 'xml' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in dataDensify: Name 'stepsize' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 24, in parseSLD_v1_0_0: Can't assign attribute 'opacity' on List[nothing] [not-writable]"
False,
True,"File ""dummpy_input_file"", line 15, in generateColorMap: No attribute 'sourceValue' on str [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 21, in main: Function getopt.getopt was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in parseLegend: Can't assign attribute 'legendentries' on Dict[nothing, nothing] [not-writable]"
False,
True,"File ""dummpy_input_file"", line 6, in is_bright: Name 'yiq' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 4, in rgb_to_hex: Name 'rgb' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in run_command: Name 'error' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in get_file_list: Name 'filepath' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in get_url: Name 'response' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in cli: Name 'cmd' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 6, in test_subscription_cancel_page: Name 'response' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in users: No attribute 'session' on List[Dict[str, Union[bool, str]]] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 9, in issues: No attribute 'session' on Dict[str, str] [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in list_commands: No attribute 'startswith' on List[nothing] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in result_to_dataframe: Name 'csv' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_pricing_page_as_subscriber: Name 'response' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in __find_thresholds: Name 'threshold' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in splitHistogramKittlerIllingworth: Name 'threshold' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in _deselectList: No attribute 'classListBox' on int [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in endpoint: Name 'instanceargs' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in errorhandle: Name 'messages' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in __depth_visitor_run: No attribute 'main_module' on bool [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 5, in add_enter_code: Name 'rewriter' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 58, in parse_code_object: unsupported operand type(s) for item retrieval: 'oparg: int' and 'oparg: int' [unsupported-operands]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 25, in get_imports_from_bytecode: Built-in function len was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 20, in createVM: Name 'validRecipe' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in isValid: Name 'pubkey' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in hasAwsKey: Name 'objs' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 6, in desc_html: Name 'fmt' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 15, in _convert_config_to_stdin: No attribute 'extend' on str [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in less_than_version: Name 'items' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in _merraRunid: Name 'runid' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 38, in download: No attribute 'write' on Iterator [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"invalid syntax (<unknown>, line 11)"
False,
True,"File ""dummpy_input_file"", line 6, in calcSeverity: Name 'nt' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in weights_filename: Name 'filename' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 9, in _validate_md5: Name 'digest' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in test_hello_handler_parse: No attribute 'parse' on List[str] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in equivalent: Name 'keys' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in compute: Name 'results' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in on_chosen_inline_result: Name 'result_id' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in _stateToDb: No attribute 'execute' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in bulletins: Name 'url' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 32, in detournotices: No attribute 'append' on datetime.datetime [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in create_multiple_workers: No attribute 'start' on int [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in get: Name 'result' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 8, in get: Name 'result' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in to_text: Name 'port' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in get_hugepage_size: Name '_' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 4, in get_hugepage_nr: Name 'stdout' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in _user: No attribute 'psql' on str [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 8, in create: Name 'command' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in find_by_code: Name 'formatted_code' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in http_delete_server_id: Name 'result' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in to_text: Name 'nic' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 14, in set: No attribute '__iter__' on None [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 13, in to_text: No attribute 'modules' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in simple_function: Name 'msg' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in terminate: Name 'e' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in serialize: No attribute 'percent_off' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 22, in cancel: Name 'cancelled' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in k: No attribute '_lock' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in database_name: Name 'info' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 4, in store_directory: Name 'info' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in _readSwe: Name 'out' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in readSnowbands: Name 'line' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 10, in readVegetation: Name 'data' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 31, in download: unsupported operand type(s) for item retrieval: 'res: float' and '0: int' [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in dates: Name 'dts' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in __getCachedSource: Name 'data' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 12, in getSource: Name 'data' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in getImage: Name 'image' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in add_if_none: Name 'rs' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in test_current_is_none_at_start: Name 'cursor' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_can_init_unmanaged_extension: Name 'plugin' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in __init__: Can't assign attribute 'key_value_separator' on Dict[str, Any] [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_cannot_move_beyond_end: Name 'cursor' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 3, in test_can_only_move_until_end: Name 'cursor' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 3, in test_moving_by_zero_keeps_same_position: Name 'cursor' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 3, in test_keys_are_populated_before_moving: Name 'cursor' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in start: No attribute 'running' on tuple [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in Array: 'array' object is not callable [not-callable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in __repr__: Function json.dumps was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in __repr__: No attribute 'content' on list [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 14, in __get_used_net_vlan: Name 'e' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 11, in get_free_net_vlan: Name 'r' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 12, in get_table: No attribute 'keys' on str [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 22, in new_tenant: No attribute 'logger' on str [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 18, in new_row: No attribute 'con' on str [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 24, in update_rows: Name 'e' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 37, in get_host: Name 'e' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 8, in new_uuid: unsupported operand type(s) for -: 'uuid: str' and '1: int' [unsupported-operands]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 16, in edit_host: Name 'e' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in _getJsonMeta: Name 'meta' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 16, in apply: Name 'item' is not defined [name-error]
File ""dummpy_input_file"", line 16, in apply: Built-in function isinstance was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in post: No attribute 'get_success_url' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in __init__: Can't assign attribute '_error_string' on Dict[str, Any] [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in _pack_context: Name 'context_d' is not defined [name-error]"
False,
False,
True,"File ""dummpy_input_file"", line 4, in reply: No attribute 'msg_id' on tuple [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in _call_novaclient: Name 'client' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in check_img_metadata_quota_limit: Name 'quota_metadata' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 14, in _adjust_child_weights: Name 'offset' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 21, in _schedule: Name 'host_dict' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in get_content_type: Name 'content_type' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in find_first_child_named: Name 'node' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in serialize: No attribute 'serialize_body' on webob.response.Response [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in get_project: Name 'project' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in get_projects: Name 'result' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in is_in_project: Name 'project' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_path_direction: Name 'path' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 15, in upgrade: Name 'instances' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 15, in get_user_roles: Name 'role' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in delete_project: Name 'project_dn' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in __find_group_dns_with_member: No attribute '__find_dns' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in get_actions: Name 'actions' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 13, in _format_security_group_rule: Name 'source_group' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in _make_agent_call: Name 'ret' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in add_to_param_xenstore: No attribute '_session' on str [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in remove_from_param_xenstore: Name 'vm_ref' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in mpi_to_dec: Name 'hexval' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in dec_to_mpi: Name 'bn' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in _action_rebuild: No attribute 'getAttribute' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in describe_user_roles: Name 'roles' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 13, in generate_x509_for_user: Name 'user' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 16, in describe_hosts: unsupported operand type(s) for item retrieval: list and str [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in _get_auth_header: Name 'ex' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in _create_local_pif: Name 'pif_ref' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in VM_remove_from_xenstore_data: Name 'db_ref' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in _logout: Name 's' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in test_service_enabled_on_create_based_on_flag: No attribute 'service_id' on str [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 7, in test_create: No attribute 'assert_' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in call_xenapi: No attribute 'split' on tuple [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 37, in update_status: Name 'e' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 7, in _parse_xmlrpc_value: Name 'x' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 8, in _handle: Name 'server' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in execute_get_output: No attribute 'stdout' on Iterator[nothing] [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in configure_logging: Function SysLogHandler.setFormatter was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in _deserialize_create: Name 'deserializer' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in _get_kernel_ramdisk_from_image: Name 'ramdisk_id' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 38, in create: Name 'exp' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 10, in _security_group_rule_exists: unsupported operand type(s) for item retrieval: 'is_duplicate: bool' and 'key: str' [unsupported-operands]"
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 24, in get_resources: No attribute 'append' on Dict[str, Any] [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in create: Name 'security_group_rule' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in _inject_key_into_fs: Name 'sshdir' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 15, in create_region: Name 'switch' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in log_db_contents: Name 'text' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in with_http_connection: Name 'conn' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 94, in delayed_create: Name 'combined' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 6, in _is_image_available: Name 'properties' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in delete_record: Name 'result' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 10, in _run_command: Name 'err' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in _handleElapsedTime: No attribute 'slow_tests' on Tuple[Any, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in __init__: Can't assign attribute 'show_elapsed' on tuple [not-writable]"
False,
True,"File ""dummpy_input_file"", line 8, in _writeSlowTests: Name 'elapsed_time' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_001_can_create_keypair: Name 'key' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 8, in test_002_security_group: Can't assign attribute 'connection' on str [not-writable]"
False,
True,"File ""dummpy_input_file"", line 15, in test_003_instance_with_group_runs_within_60_seconds: Name 'ip' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 15, in __init__: No attribute 'supported' on TextIO [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in _call: unsupported operand type(s) for item retrieval: str and str [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in reset_network: Name 'resp' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 28, in __call__: Name 'detail' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in inner: No attribute 'connect' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in get_project: Name 'dn' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_extended_action: No attribute 'body' on str [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 5, in test_invalid_action_body: No attribute 'assertEqual' on Dict[str, Dict[str, str]] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 5, in test_invalid_action: Name 'response' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 14, in test_get_resources_with_mgr: unsupported operand type(s) for item retrieval: 'request: webob.request.Request' and ''big_bands': str' [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in cleanup_shared_storage_test_file: Name 'tmp_file' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in generate_fingerprint: Name 'err' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in revoke_certs_by_project: Name 'cert' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in create_vdi: Name 'vdi_ref' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in _check_vdi_size: Name 'size_bytes' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in determine_from_glance: Name 'image_id' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in preconfigure_instance: Name 'metadata' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in get_vhd_parent: Name 'parent_ref' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in get_net_and_prefixlen: No attribute '_prefixlen' on netaddr.ip.IPNetwork [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 4, in get_ip_version: Name 'net' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in test_get_flavor_by_invalid_id: No attribute 'assertEqual' on webob.response.Response [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in test_detail: Name 'output' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_can_delete_user: Name 'userlist' is not defined [name-error]"
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in _shutdown_instance: Name 'network_info' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in confirm_resize: Name 'usage_info' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in list_from_xenstore: Name 'ret' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in __init__: Can't assign attribute 'project' on Dict[str, Any] [not-writable]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in __init__: No attribute 'create_user' on Dict[str, str] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_bad_status_value: No attribute 'assertRaises' on Dict[str, str] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in _calc_signature_0: unsupported operand type(s) for +: List[nothing] and str [unsupported-operands]"
False,
True,"File ""dummpy_input_file"", line 25, in _calc_signature_2: No attribute 'append' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 21, in init_host: Name 'expect_running' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in _create_group: Name 'values' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 14, in test_create_instance_with_img_ref_associates_config_drive: Name 'instance' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 8, in test_default_hostname_generator: No attribute 'assertEqual' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_start: Name 'instance_id' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_set_admin_password: Name 'instance_id' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_properties_root_defice_name: Name 'properties0' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_get_all_security_groups: Name 'rv' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 13, in test_create_delete_security_group: No attribute 'expect_http' on str [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 7, in test_group_name_valid_chars_security_group: No attribute 'assertRaises' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 18, in test_get_instance_disk_info_works_correctly: No attribute 'mox' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 2, in XMLDesc: No attribute '_fake_dom_xml' on tuple [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in test_xml_and_uri_no_ramdisk_no_kernel: Name 'instance_data' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 4, in test_xml_and_uri_no_ramdisk: No attribute '_check_xml_and_uri' on Dict[str, str] [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in test_lxc_container_and_uri: Name 'instance_data' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 2, in __init__: Can't assign attribute '__dirty__' on Dict[str, Any] [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 20, in __init__: Name 'model_type' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_index_json: Name 'request' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in test_index_json: Name 'response' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 7, in setUp: No attribute '_empty_app' on str [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in test_limited_request_xml: No attribute 'assertEqual' on webob.response.Response [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_delay_GET: No attribute '_check' on List[None] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 5, in test_multiple_delays: No attribute 'assertEqual' on list [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in test_index: Name 'actual' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in test_index_xml: Name 'response' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 9, in test_index_nonexistant_server: No attribute 'assertEqual' on webob.response.Response [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 21, in test_create: No attribute 'assertEqual' on Dict[str, Dict[str, str]] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 21, in test_create_xml: No attribute 'assertEqual' on webob.response.Response [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 11, in test_create_empty_body: No attribute 'assertEqual' on webob.response.Response [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in _is_vdi_pv: Function re.search was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in list_instances_detail: Name 'vm_rec' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 21, in _get_vm_opaque_ref: Name 'instance_name' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in get_resources: No attribute 'resource_ext' on List[nothing] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 4, in get_actions: No attribute 'action_ext' on List[nothing] [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in setUp: No attribute 'ext_list' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in setUp: No attribute 'flags' on str [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 16, in test_get_resources_with_controller: Name 'response' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in test_get_resources: Name 'ext_midware' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 7, in test_invalid_extensions: Name 'ext_midware' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in test_create_instance_with_network_no_id: Name 'compute_api' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 4, in test_create_instance_with_network_no_id_xml: Name 'body_dict' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in _parse_db_block_device_mapping: unsupported operand type(s) for item assignment: Tuple[str, str, str, str, str, str, str] [unsupported-operands]"
False,
True,"File ""dummpy_input_file"", line 9, in test_update_block_device_mapping: Name 'instance_type' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 5, in test_volume_size: Name 'inst_type' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_instance_get_project_vpn: Name 'instance' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 7, in test_instance_get_project_vpn_joins: No attribute 'context' on Dict[str, Any] [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 14, in test_instance_get_all_by_filters_deleted: unsupported operand type(s) for item retrieval: Dict[str, Union[int, str]] and int [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in test_host_startup: Name 'result' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in __init__: No attribute '__mail_proc' on tuple [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 22, in _create_disk: Name 'new_resources' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in _clone_wmi_obj: Name 'newinst' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 22, in get_info: No attribute 'MemoryUsage' on list [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 16, in handle: No attribute 'start' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in human_status: Name 'extras' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in wrapper: No attribute 'create_object' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in create_object: unsupported operand type(s) for item retrieval: str and str [unsupported-operands]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in path_parts: Name 'path' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in short_date: Name 'df' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 5, in iso_date: Name 'df' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 23, in call_win32: Name 'kwargs' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 14, in test_update_all_malformed_container: Name 'res' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_get_servers: Name 'servers' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in test_namespace_limits: Name 'response' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_empty_domain: Name 'result' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_create_instance_with_network_no_fixed_ip_xml: Name 'compute_api' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 13, in test_create_instance_with_security_group_json: No attribute 'status_int' on List[str] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 13, in test_get_server_by_id_verify_security_groups_json: Name 'res_dict' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 17, in test_get_server_by_id_verify_security_groups_xml: Name 'sec_group' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_snapshot: No attribute 'context' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_ajax_console: Name 'console' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in test_too_many_metadata_items_on_create: Can't assign attribute 'method' on int [not-writable]"
False,
True,"File ""dummpy_input_file"", line 12, in test_too_many_metadata_items_on_update_item: Name 'res' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in _get_image_properties: No attribute 'get' on float [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 21, in _copy_vmdk_content: Name 'copy_disk_task' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in _get_datacenter_name_and_ref: Name 'dc_obj' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 6, in _path_exists: Name 'search_task' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in _get_vm_ref_from_the_name: Name 'vm' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in decorated: Name 'key' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in save: No attribute 'title' on tuple [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 4, in save: No attribute 'slug' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in _pollEvent: Name 'resource' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in fromCertificateAndKeyPair: Name 'privcert' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 15, in test_multinic_iptables: Name 'ipv4_network_rules' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in setUp: No attribute 'user_id' on Type[Mock] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in get_object_tags: Name 'objParts' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in tag_exists: Name 'o' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 3, in ns_exists: Name 'status' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 31, in get_typed_tag_value: Name 'r' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_iterconstants: No attribute 'STATUS' on list [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 5, in test_attributeIterconstantsIdentity: No attribute 'assertIdentical' on list [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_iterconstants: No attribute 'assertEqual' on list [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 6, in test_attributeIterconstantsIdentity: No attribute 'FXF' on list [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 5, in test_iterconstantsIdentity: Name 'again' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_name: Name 'flag' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 4, in test_representation: Name 'flag' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_representation: Name 'writeAppend' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in inspect: No attribute 'original' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in __getstate__: Name 'd' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 21, in __init__: Function base64.b64encode was called with the wrong arguments [wrong-arg-types]"
False,
True,"File ""dummpy_input_file"", line 22, in get_of_switches: Name 'e' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 12, in obtain_port_correspondence: No attribute 'json' on str [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 22, in new_flow: No attribute 'pp2ofi' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in convert_datetime2str: Name 'k' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 23, in new_flavor: No attribute 'logger' on int [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 21, in new_image: Name 'e' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 29, in get_instance: unsupported operand type(s) for item retrieval: List[nothing] and str [unsupported-operands]"
False,
True,"File ""dummpy_input_file"", line 28, in new_instance: Name 'e' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 11, in get_ports: Name 'e' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in get_os_information: No attribute 'read' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in removeNegotiator: Name 'oldNegotiator' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 3, in getNegotiator: Name 'negotiatorKey' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in __new__: No attribute 'iteritems' on Dict[nothing, nothing] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 13, in __init__: No attribute 'commandName' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in __repr__: No attribute 'isLocal' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 15, in options: No attribute 'update' on tuple [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in __init__: Can't assign attribute 'certificate' on Dict[str, Any] [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 16, in __init__: Can't assign attribute 'listeners' on List[int] [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 23, in addUTMPEntry: Can't assign attribute 'ut_id' on int [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 15, in closed: No attribute 'pty' on int [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 17, in testShifting: No attribute 'term' on int [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in testEraseToLineEnd: No attribute 'term' on str [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in testEraseToDisplayEnd: No attribute 'assertEqual' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in fromBox: Name 'st' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in set_group_readable: Name 'fullgroup' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in rm_r: Name 'si' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in getChallenge: Name 'o' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 9, in decode: Name 'auth' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in validQuery: No attribute '_ebLookup' on Tuple[Any, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in test_deprecatedPreservesName: Name 'dummy' is not defined [name-error]
File ""dummpy_input_file"", line 11, in test_deprecatedPreservesName: 'deprecated' object is not callable [not-callable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_iterconstants: No attribute 'assertEqual' on list [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 6, in test_attributeIterconstantsIdentity: No attribute 'METHOD' on list [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_versionMetadata: 'deprecated' object is not callable [not-callable]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_deprecatedReplacement: 'deprecated' object is not callable [not-callable]"
False,
True,"File ""dummpy_input_file"", line 10, in test_deprecatedReplacementWithCallable: Name 'dummy' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in simpleModuleEntry: Name 'paths' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in test_filteredWarning: Name 'warningsShown' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_recordIndexable: Name 'password' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in testUniq: No attribute 'assertEqual' on List[Union[int, str]] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in test_syscallError: Name 'exc' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in connectionReady: Name 'notifier' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in cachedRemotelyAs: Name 'luid' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in proto_answer: Name 'd' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in proto_decref: Name 'refs' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in descrFromDoc: Name 'lines' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in __repr__: Name 'state' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in __getattribute__: Name 'state' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in __getstate__: Name 'state' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in needRebuildUpdate: Name 'yn' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in __getattr__: Name 'result' is not defined [name-error]"
